src/github.com/stellar/horizon/config.go

```diff
@@ -32,4 +31,9 @@ type Config struct {
 	// determining a "retention duration", each ledger roughly corresponds to 10
 	// seconds of real time.
 	HistoryRetentionCount uint
+
+	// StaleThreshold represents the number of ledgers a history database may be
+	// out-of-date by before horizon begins to respond with an error to history
+	// requests.
+	StaleThreshold uint
```

Comment:
Why not HistoryStaleThreshold as it is about the history.
it's there in src/github.com/stellar/horizon/cmd/horizon/main.go:50